plugins/python-build/share/python-build/pyston-0.5.1-numpy

```diff
@@ -0,0 +1 @@
+source "${BASH_SOURCE%/*}/pyston-0.5.1"
```

Comment:
Done in https://github.com/yyuu/pyenv/pull/718/commits/8acb4ebaba1010112fd6b48c3c2311894b98ffc9